test/auth/legacy/connection-string.json

```diff
@@ -440,6 +440,21 @@
         }
       }
     },
+    {
```

Comment:
This will need to be skipped, we're blocked on [PYTHON-5445](https://jira.mongodb.org/browse/PYTHON-5445)